subworkflows/nf-core/bam_tumor_only_somatic_variant_calling_gatk/main.nf

```diff
@@ -20,94 +18,75 @@ workflow BAM_TUMOR_ONLY_SOMATIC_VARIANT_CALLING_GATK {
     ch_interval_file         // channel: /path/to/interval/file
 
     main:
-    ch_versions = Channel.empty()
+    versions = Channel.empty()
 
-    //
     //Perform variant calling using mutect2 module in tumor single mode.
-    //
-    MUTECT2 (
+    GATK4_MUTECT2(
         ch_input,
         ch_fasta,
         ch_fai,
         ch_dict,
         ch_germline_resource,
         ch_germline_resource_tbi,
         ch_panel_of_normals,
-        ch_panel_of_normals_tbi
+        ch_panel_of_normals_tbi,
```

Comment:
```suggestion
        ch_panel_of_normals_tbi
```

Reply:
That's coming from the language server, it's valid

Reply:
A comma with no further output is valid?
Might not give an error, but doesn't make it valid.
Smells like a bug in the language server.